src/witan/cic/episodes.clj

```diff
@@ -711,15 +750,19 @@
 
    ;; Group by ID and process each group
    headers-and-episodes-by-id-xf
+
+   ;; add the header
+   create-header-xf
+   add-header-fields-to-episodes-xf
+
+   ;; Handle bad episode dates
    mark-stale-episodes-xf
    mark-overlapping-episodes-for-id-xf
    mark-episode-overlapped-by-open-episodes-xf
+   (mark-stale-history-xf max-report-year)
    mark-ordered-disjoint-episodes-xf
 
    ;; Things that want a clean list of episodes
    separate-episodes-for-removal-xf ;; make the clean list
    add-periods-of-care-xf
```

Comment:
What is line 751 to 767?

Reply:
not sure what you mean here

Reply:
ah I expanded the code above this section and saw the `comp` at the beginning, before that it just looked like a list of fns not actually be called, just like it was text/a comment. Apologies